requirements-dev.txt

```diff
@@ -0,0 +1,16 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+# Common dependencies for code quality control (testing, linting, static checks, etc.) ---------------------------------
+
+flake8==3.9.2
+flake8-tidy-imports==4.3.0
+isort[colors]~=5.8.0
+mock==4.0.3
+mypy==0.812
```

Comment:
I think it could be problematic for databuilder given it is used for library not a service like other modules, we should at least make it as range instead of pin down to certain versions.

Reply:
ok, so `>=` for all requirements dev ?

Reply:
let's make it range .